src/core/resolvers/arco.ts

```diff
@@ -167,6 +167,12 @@ export type AllowResolveIconOption = true | { enable: true; iconPrefix?: string
 export type ResolveIconsOption = DisallowResolveIconOption | AllowResolveIconOption
 
 export interface ArcoResolverOptions {
+  /**
+   * exclude components that do not require automatic import
+   *
+   * @default []
+   */
+    exclude?: string[]
```

Comment:
```suggestion
    exclude?: (string | RegExp)[]
```

I think it makes sense to support regex as well